Makefile

```diff
@@ -0,0 +1,52 @@
+.DEFAULT_GOAL := all
+
+DOCKER_EXTRA_ARGS ?=
+
+export DOCKER_BUILDKIT ?= 1
+
+define BUILDER
+	docker container run \
+		--env CCACHE_DIR=/openadr-ven-library/.ccache \
+		--init \
+		--rm \
+		--volume "$(shell pwd)":/openadr-ven-library/ \
+		--user $(shell id -u):$(shell id -g) \
+		--workdir /openadr-ven-library/ \
+		$(DOCKER_EXTRA_ARGS) \
+		analyticsfire/openadr-ven-library-builder
+endef
+
+.PHONY: all
+all: build
+
+.PHONY: build
+build: builder
+	$(BUILDER) cmake -B build -DTEST=1 -DCOVERAGE=1
```

Comment:
It would be nice to have a binary produced which doesn't pay overload of coverage in runtime.

Reply:
I can enable code coverage by variable in Makefile e.g.,:

```
COVERAGE ?= 0
...
$(BUILDER) cmake -B build -DTEST=1 -DCOVERAGE=${COVERAGE}
```

and next we can run it via:

```
COVERAGE=1 make build
# or
make build -e COVERAGE=1
```

What do you think?

Reply:
Yes, that looks great! When we publish build artefacts, we will most likely be interested in version COVERAGE=0.

Reply:
Added in 54ced47